smart_open/tests/test_hdfs.py

```diff
@@ -31,23 +32,29 @@
 # such as Windows.
 #
 class CliRawInputBaseTest(unittest.TestCase):
+    def _setUp(self, test_file):
+        if platform == "linux" or platform == "linux2":
+            path = P.join(CURR_DIR, 'test_data/' + test_file)
+            self.cat = subprocess.Popen(['cat', path], stdout=subprocess.PIPE)
+        elif platform == "win32":
```

Comment:
Other platforms (e.g. MacOS aka darwin) have cat. I suggest handling Windows as a special case, and expecting cat to be there otherwise.